HW_5/26.py

```diff
@@ -0,0 +1,10 @@
+A = int(input("Введите число: "))
+B = int(input("Введите степень числа: "))
+def power(a, b):
+    if b == 0:
+        return 1
+    elif b == 1:
+        return a
+    else:
+        return a * power(a, b - 1)
```

Comment:
Первое условие не нужно так как есть второе, этот базовый случай наступит раньше. А так как внутри второго условия есть `return`, то `else` не нужен.
```suggestion
    if b == 1:
        return a
    return a * power(a, b - 1)
```